files/class-image.php

```diff
@@ -71,7 +73,8 @@ public function resize( $size_data ) {
 
 		$this->set_width_height( $dimensions );
 
-		return $this->is_resized = true;
+		$this->is_resized = true;
+		return true;
```

Comment:
Since we already assign `$this->is_resized` to `true` earlier and that's the same value we want to return, might be worth directly returning that instead:
```suggestion
		return $this->is_resized;
```

Reply:
Fixed in 34509a2ca